src/components/profile/ProfileService.js

```diff
@@ -266,7 +302,35 @@ goog.require('ga_urlutils_service');
               .attr('dy', '1em')
               .text($translate.instant(options.yLabel) + ' [m]');
 
-           return element;
+          //Total Elevation Difference
+          group.append('text')
+              .attr('class', 'ga-profile-label ga-profile-elevation-difference')
+              .attr('x', 0)
+              .attr('y', height + options.margin.bottom - 2)
+              .style('text-anchor', 'left')
+              .attr('font-size', '0.9em')
+              .text('\u03A3 \u0394 ' + $translate.instant(options.yLabel) +
+                  ': ' + Math.round(this.diff * 100) / 100 + ' m');
```

Comment:
To use the number formating suggested by @gjn, have a look to the `measure` filter [here](https://github.com/geoadmin/mf-geoadmin3/blob/kom_elevations/src/components/measure/MeasureService.js#L7)